components/usage/pkg/db/workspace_instance_usage.go

```diff
@@ -89,3 +89,5 @@ func ListUsage(ctx context.Context, conn *gorm.DB, attributionId AttributionID,
 	}
 	return usageRecords, nil
 }
+
+type UsageReport []WorkspaceInstanceUsage
```

Comment:
I think it would be good also to store `StartTime`, `EndTime`, and `GenerationTime` in this shape at some point.

Reply:
Great point! Will add in next PR.

Reply:
https://github.com/gitpod-io/gitpod/pull/12503 to do this.